scripts/consume_sitemaps.py

```diff
@@ -17,7 +19,9 @@
 from isb_web import sqlmodel_database
 from isb_web.sqlmodel_database import SQLModelDAO
 
-CONCURRENT_DOWNLOADS = 20
+CONCURRENT_DOWNLOADS = 50
+# when we hit this length, add some more to the queue
+REFETCH_LENGTH = CONCURRENT_DOWNLOADS / 2
```

Comment:
Introduce a new variable to kick off prefetching of new records while the current ones are being processed.